pkg/registry/core/endpoint/storage/storage.go

```diff
@@ -59,3 +68,39 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"ep"}
 }
+
+// Get gets endpoints resource by name in the context (whose significance is tenant and namespace).
+// If the target name starts with prefix of "kubernetes-", and the contextual namespace is "default",
+// this target is considered as "k8s alias", a read-only copy of kubernetes endpoints of default namespace
+// in system tenant, and its content returned is based on the system kubernetes resource.
+func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
+	tenant, ok := genericapirequest.TenantFrom(ctx)
+	if !ok {
+		tenant = "system"
```

Comment:
Please use metav1.TenantSystem and metav1.NamespaceDefault  in place of "system" and "default" throughout the code.

Reply:
yes it makes sense to use const instead of arbitrary string literals here. will update as suggested.